Documentation/storage.md

```diff
@@ -0,0 +1,161 @@
+# Storage options
+
+Dex requires persisting state to perform various tasks such as track refresh tokens, preventing replays, and rotating keys. This document is a summary of the storage configurations supported by dex.
+
+Regardless of the implementation, dex saves sensitive data in its backing storage: signing keys, bcrypt'd passwords, etc. Storage breaches are serious because attacks may propagate to any application which relies on dex. Whatever storage option used, transport security and database ACLs are both musts.
```

Comment:
Ah.  I think I read the thing about signing keys and didn't read anything about how to set that up (because it hasn't been ported yet). That's not required if this is the only doc that covers those topics.